backend/test_firestore.go

```diff
@@ -53,15 +73,56 @@ func GetData(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(user)
 }
 
-func DeleteData(w http.ResponseWriter, r *http.Request) {
+func GetEvent(w http.ResponseWriter, r *http.Request) {
+	// return json
+	w.Header().Set("Content-type", "application/json")
+
+	vars := mux.Vars(r)
+	if vars["id"] == "" {
+		w.WriteHeader(http.StatusBadRequest)
+		log.Warnf("No event ID provided")
+		return
+	}
+
+	event := &firebase.Event{FirebaseID: vars["id"]}
+	err := event.Get()
+	if err != nil {
+		w.WriteHeader(http.StatusNotFound)
+		log.Warnf("failed to get event : %v", event)
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
+	// marshal the data struct to JSON to send as response
+	json.NewEncoder(w).Encode(event)
+}
+
+func DeleteUser(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	if vars["id"] == "" {
 		log.Fatal("no document id provided")
 	}
 	user := &firebase.User{FirebaseID: vars["id"]}
 	err := user.Delete()
 	if err != nil {
-		log.Fatalf("failed to delete user : %v", user)
+		log.Fatalf("failed to delete user : %v", err)
+	}
+	w.WriteHeader(http.StatusNoContent)
+}
+
+func DeleteEvent(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	if vars["id"] == "1" {
+		w.WriteHeader(http.StatusBadRequest)
+		log.Warnf("no document id provided")
+		w.Write([]byte("No Event ID Provided"))
```

Comment:
we should write a JSON here instead of just raw bytes to prevent frontend from having to handle both JSON and non-JSON responses

Reply:
Going to just remove this since we aren't going to be sending anything back when handling errors.